controllers/drcluster_controller_test.go

```diff
@@ -409,7 +431,7 @@ var _ = Describe("DRClusterController", func() {
 			It("reports NOT validated with reason s3ListFailed", func() {
 				By("modifying a DRCluster with an invalid S3Profile that fails listing")
 				drcluster.Spec.S3ProfileName = s3Profiles[4].S3ProfileName
-				Expect(k8sClient.Update(context.TODO(), drcluster)).To(Succeed())
+				drcluster := updateDRClusterParameters(drcluster)
```

Comment:
Why do you create a new drcluster shadowing the drcluster defined in line 408?

This creates confusion when in the next steps of the test. If our intent is
to always use a new DRCluster instance for every test, we should use a different
name. The old code took this approach, calling it latestdrc.

Reply:
We want to modify the drcluster, thanks. Fixed